src/openapi.yaml

```diff
@@ -40,7 +40,7 @@ paths:
                   description: '`User.Username`, normally the same as `User.email`'
                 password:
                   type: string
-                  description: wip are there no restrictions to this password on setup such as minimum 8 characters
+                  description: wip password restrictions? minLength? charset?
```

Comment:
password currently has no restriction HOWEVER it would be good to make a minimum length of 10 characters with no other restrictions. Let's put that in the API and I will make the change on the server.